src/cpp/bindings/swigBuffer.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2014-2020 Neueda
+ */
+#pragma once
+
+namespace neueda {
+    struct SwigBuffer {
```

Comment:
There is a cross over here with https://github.com/blu-corner/codec/blob/master/src/bindings/codecBuffer.h

With this change you might be able to remove the Buffer class from the codec project. I think codec depends on sbf

Reply:
Hi Phil,

Apologies for the delay. Are you happy for me to change this SwigBuffer to match the codec one exactly updating the uses, then once merged update codec to include and use swigBuffer.h? I will rename codecBuffer.h to be decodeResults.h leaving just the DecodeResults class in it.

Thanks,
James

Reply:
So long as codec depends on sbf then that will clean-up the swig bindings in terms of managing buffers. Otherwise each project is going to end up managing its own buffer swig wrapper when it all derives from sbf anyway.

Reply:
Hi Phil,

Apologies for the delay. After looking at the codec class and adapting sbf to be similar I've realised sbf doesn't even need the additional class so I've removed it. I think your comment is still valid about having a common class, but my opinion now is a change in codec should be made to use the SbfBuffer class, what do you think?

Thanks,
James

Reply:
I've got https://github.com/james-rank/sbf/pull/1 ready to go too once this change is merged.